subsys/pm/Kconfig

```diff
@@ -37,6 +37,13 @@ config PM_S2RAM
 	help
 	  This option enables suspend-to-RAM (S2RAM).
 
+config PM_NEED_ALL_DEVICES_IDLE
+	bool "System Low Power Mode Needs All Devices Idle"
+	default n
```

Comment:
`depends on PM_DEVICE`  then you can simplify the ifdef logic below.

Reply:
also, the way it is implemented is not smp safe. So, it should also have
`depends on !SMP`

Reply:
done.